File: CSharpPractice/LeetcodePractice/Leetcode2419.cs
public class Solution2419{
    public int LongestSubarray(int[] nums) {
        int maxVal = nums.Max();
        int ans=0;
        int temp = 0;
        foreach (var num in nums)
        {
            if (num==maxVal)
            {
                temp++;
                ans= Math.Max(ans,temp);
            }
            else
            {
                temp=0;
            }
        }
        return ans;
    }
}

File: JavaScriptPractice/Leetcode2419.js
var longestSubarray = function(nums) 
{
    let maxVal = Math.max(...nums)
    let ans=0;
    let temp=0
    for(let num of nums)
    {
        if (num==maxVal) {
            temp++
            ans=Math.max(ans,temp)
        }
        else 
        {
            temp=0
        }
    }
    return ans
};